Assets/Scripts/Tutorial.cs

```diff
@@ -37,16 +38,17 @@ public enum TutorialButtonKey { Left, Up, Right };
             Destroy(tutorialRightButtonImage.gameObject);
             return;
         }
-        
-        tutorialCurrentStep = 0;
-        ShowNextButtonHint();
     }
 
     public static Tutorial GetSingleton() {
         return singleton;
     }
 
-    public bool IsInTutorial() {
+    public bool IsTutorialStarted() {
+        return isTutorialStarted;
+    }
+
+    public bool IsTutorialEnded() {
         return !isTutorialEnded;
```

Comment:
This function's name can be quite misleading, since it is actually checking the opposite (whether the tutorial has **not** ended).

I suggest changing this line to `return isTutorialEnded` (no negation), then changing all of the callers to invert their calls?